polypomo

```diff
@@ -132,6 +137,37 @@ class Status:
             self.status = "work"
             self.timer = Timer(self.worktime)
 
+    def save(self):
+        if self.status == "work":
+            with setup_conn(self.saveto) as conn:
+                if self.active:
+                    conn.execute("INSERT INTO sessions VALUES (date('now'), time('now'), NULL)")
+                else:
+                    conn.execute("""UPDATE sessions
+                                     SET stop = time('now')
+                                     WHERE start IN (
+                                         SELECT start FROM sessions
+                                         ORDER BY start DESC
+                                         LIMIT 1)
```

Comment:
```suggestion
                                     SET stop = time('now')
                                     WHERE stop is NULL
                                     ORDER BY start DESC
                                     LIMIT 1)
```

Can you check if this alternative query works?

Reply:
this query doesn't work
```
Parse error: near "order": syntax error
  essions set stop = time('now') where stop is null order by start desc limit 1;
                                      error here ---^
```

Reply:
Seems like it's a limitation of SQLite - https://stackoverflow.com/a/29074519
Thanks for trying it out.